typings/blockly.d.ts

```diff
@@ -6635,6 +6635,31 @@ declare module Blockly.Generator {
      * Category to separate generated function names from variables and procedures.
      */
     var NAME_TYPE: any /*missing*/;
+
+    /**
+     * instance of javascript generator
+     */
+     var javascript: Blockly.Generator;
```

Comment:
Can you move these instances into the generator typings file itself?

That way if one only includes some generators but not others, they don't get them all

Reply:
Done